src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -113,7 +113,7 @@
             <argument type="service" id="api_platform.item_data_provider" on-invalid="ignore" />
             <argument>%api_platform.allow_plain_identifiers%</argument>
             <argument>null</argument>
-            <argument type="tagged" tag="api_platform.data_transformer" on-invalid="ignore" />
+            <argument type="tagged_iterator" tag="api_platform.data_transformer" on-invalid="ignore" />
```

Comment:
Changing this would break compatibility with Symfony < 4.4

Bad... :disappointed:

Reply:
This one is trickier, the only solution I see is to set the appropriate type in the extension.

Reply:
I suggest we can make this change in `master` (since ugly / hacky workarounds would be worse...), but first we should target `2.5` for Symfony 4.4 compatibility.